core/data/core.joke

```diff
@@ -4538,3 +4538,31 @@
   ([] (exit 0))
   ([^Int code]
    (exit__ code)))
+
+(defn ^BigInt precision
```

Comment:
It feels like `precision` and `set-precision` belong in `joker.math`. Would you agree?

Reply:
I wrestled with this question. I expect anyone using these functions for "real work" would be pulling in `joker.math` anyway, so that isn't a concern here; but, since these functions pertain to the builtin (one might say "core") `BigFloat` type, they seem kinda-core-ish to me, as are `+` and many other math functions that are nevertheless fundamental.

On the other hand, anything Joker introduces to `joker.core` risks someday conflicting with whatever might go into `clojure.core`.

Since I don't have a strong opinion, I'll go with whichever you like. Just let me know!

Reply:
My approach is to not put anything in the core namespace unless it's obvious it belongs there. Let's move these two functions into joker.math.

Reply:
> My approach is to not put anything in the core namespace unless it's obvious it belongs there. Let's move these two functions into joker.math.

Sounds good...done, though it was more involved (due to trying to keep/make things clean) than I expected when I plunged into this.